frontend/public/style/_overrides.scss

```diff
@@ -160,6 +160,7 @@ kbd {
 // specificity targeting form elements to override --pf-global--FontSize--md
 .pf-c-page,
 .modal-dialog {
+  .pf-c-app-launcher,
```

Comment:
@rhamilto is this still needed now that we changed this globally? If not, we should try to clean it up.